packages/driver/cypress.config.ts

```diff
@@ -20,4 +21,14 @@ export default defineConfig({
     },
     baseUrl: 'http://localhost:3500',
   },
+  component: {
+    specPattern: 'cypress/component/**/*.cy.js',
+    supportFile: false,
+    devServer: (devServerOptions) => {
```

Comment:
Any reason to use the function over the object API?

Reply:
This package (driver) doesn't have any component tests (there aren't any components to be tested). 

If you use the object API, you need to specify `framework` - and we don't have one here.

Unless it's possible to just use `bundler` without `framework`? I don't think we support this, do we?

Reply:
The types might not be happy but you should be able to do:
```js
devServer: {
  bundler: 'webpack',
  framework: undefined
} as any
```